pkg/api/v1/task.go

```diff
@@ -121,6 +125,16 @@ func (g *TaskGroup) addOutputsToTask(ctx context.Context, workspaceName string,
 	return nil
 }
 
+func (g *TaskGroup) addStatsToTask(ctx context.Context, workspaceName string, task *types.TaskWithRelated) error {
+	tasksInFlight, err := g.taskRepo.TasksInFlight(ctx, workspaceName, task.Stub.ExternalId)
+	if err != nil {
+		return err
+	}
+
+	task.Stats.InFlight = uint32(tasksInFlight)
```

Comment:
In the V1 dashboard, we called this queue depth. Not sure if you like that more than this

Reply:
Yeah good thought. @mernit do you have a preference?

Reply:
Renamed to queue_depth.